### PR TITLE
Update default max messages in manage-mysql.md

### LINKED ISSUE #3978: Feedback on page: /guides/v2.3/config-guide/mq/manage-mysql.html
# General issue

Topic about Managing message queues is confusing: 

Why is the page about Managing message queues named `manage-*mysql*.html?`.  The preamble is confusing b/c the cloud solution can (and recommends) rabbitmq and cron. (The topic is written as if it's one or the other.) Also, this doc says the default max messages is 10k in one place and then 1k in another and then 0 in another. On Cloud it is 1K.

Also, Magento Commerce Cloud customers can use the CRON_CONSUMER_RUNNER variable to manage message queues, and that's not mentioned here.

### PR DESCRIPTION
**Pull request by RubanSvitlana**
Fixes #3978 

## This PR is a:
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary
When this pull request is merged, it will fix the default number of max messages, which is 1000 (reference: https://devdocs.magento.com/guides/v2.3/cloud/env/variables-deploy.html) instead of 10000. 

## Additional information

List all affected URLs 
- https://devdocs.magento.com/guides/v2.3/config-guide/mq/manage-mysql.html
- https://devdocs.magento.com/guides/v2.2/config-guide/mq/manage-mysql.html

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4012) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4012) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@RubanSvitlana thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by RubanSvitlana** [comment]
Can I update this PR with the change of the name to manage-message-queues.md also?

File: guides/v2.2/config-guide/mq/manage-mysql.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ You can also use a process manager such as [Supervisor](http://supervisord.org/i
 
 * Cron job `consumers_runner` is enabled
 * Cron job `consumers_runner` runs all defined consumers
-* Each consumer process 10000 messages and then terminate
+* Each consumer process 1000 messages and then terminate
```
Comment (meker12) [inline guides/v2.2/config-guide/mq/manage-mysql.md]:
```suggestion
* Each consumer processes 1000 messages and then terminates
```

File: guides/v2.3/config-guide/mq/manage-message-queues.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ You can also use a process manager such as [Supervisor](http://supervisord.org/i
 
 * Cron job `consumers_runner` is enabled
 * Cron job `consumers_runner` runs all defined consumers
-* Each consumer process 10000 messages and then terminate
+* Each consumer process 1000 messages and then terminate
```
Comment (meker12) [inline guides/v2.3/config-guide/mq/manage-message-queues.md]:
```suggestion
* Each consumer processes 1000 messages and then terminates
```

**Comment by RubanSvitlana** [comment]
Hello, @meker12 !
I've updated the v2.2 name and added redirect within this PR too.
Thanks :)

**Comment by shrielenee** [comment]
@meker12 Any news on this? Looks like some changes were completed.

**Comment by meker12** [comment]
running tests

**Comment by meker12** [comment]
running tests

**Pull request merged**